databricks_cli/dbfs/cli.py

```diff
@@ -87,26 +87,6 @@ def rm_cli(recursive, dbfs_path):
     delete(dbfs_path, recursive)
 
 
-def copy_to_dbfs(src, dbfs_path_dst, overwrite):
-    try:
-        put_file(src, dbfs_path_dst, overwrite)
-    except HTTPError as e:
-        if e.response.json()['error_code'] == DbfsErrorCodes.RESOURCE_ALREADY_EXISTS:
-            click.echo(e.response.json())
-        else:
-            raise e
-
-
-def copy_from_dbfs(dbfs_path_src, dst, overwrite):
-    # Will truncate the local file if it exists.
-    try:
-        get_file(dbfs_path_src, dst, overwrite)
-    except LocalFileExistsException:
-        click.echo(('The host destination {} already exists. You should provide the '
-                   '--overwrite flag.')
```

Comment:
The `--overwrite` hint is missing w/ the new code, which could be quite helpful.